pushkin/sender/sender_manager.py

```diff
@@ -8,18 +8,36 @@
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 '''
+import sys
+
 from pushkin.sender.nordifier import constants
-from .senders import ApnNotificationSender
-from .senders import GcmNotificationSender
-from .senders import NotificationPostProcessor
+from pushkin.sender import senders
 from pushkin import context
+from pushkin import config
 
 
 class NotificationSenderManager():
     def __init__(self):
-        self.apn_sender_processor = ApnNotificationSender()
-        self.gcm_sender_processor = GcmNotificationSender()
-        self.notification_post_processor = NotificationPostProcessor()
+        self.apn_sender_processor = senders.ApnNotificationSender()
+        self.gcm_sender_processor = senders.GcmNotificationSender()
+        if config.ext_sender_enabled:
+            # NOTE: Load custom implementation of notification sender
+            #       (must inherit from senders.NotificationSender)
+            try:
+                mod_name, cls_name = config.ext_sender.rsplit('.', 1)
```

Comment:
It seems to be not good idea to modify constants in run-time, so I'm going to remove PLATFORM_BY_PROVIDER from constants at all